src/d3d8/d3d8_options.cpp

```diff
@@ -0,0 +1,55 @@
+#include "d3d8_options.h"
+#include "../d3d9/d3d9_bridge.h"
+#include "../util/config/config.h"
+#include "../util/util_string.h"
+
+#include <charconv>
+
+namespace dxvk {
+  static inline uint32_t parseDword(std::string_view str) {
```

Comment:
Don't we already have stuff for this in regular options stuff?

Reply:
This is needed for `d3d8.forceVsDecl` which is an array of DWORD